.circleci/common.sh

```diff
@@ -92,27 +92,6 @@ function install_deps_pytorch_xla() {
 
   sudo ln -s "$(command -v bazelisk)" /usr/bin/bazel
 
-  # Install gcc-11
```

Comment:
I wonder why this section is removed

Reply:
Previously, install gcc-11 from setup script. With this update, we are using gcc-10 that is standard in debian:bullseye, so we don't need to install gcc again in our setup script